xbmc/music/MusicDatabase.cpp

```diff
@@ -13234,6 +13234,13 @@ int CMusicDatabase::GetOrderFilter(const std::string& type,
     }
   }
 
+  // Get the right tableview as if we are using strArtistSort the column name is ambiguous
+  std::string table{};
```

Comment:
You can omit the `{}` as `std::string`default ctor will be called automatically.

Reply:
OK. Thanks.  Just wanted to make sure it would be an empty string.